docs/superpowers/plans/2026-04-30-pipette-followups.md

```
@@ -0,0 +1,1717 @@
+     --event=grill_symbol_verified --data=symbol=<the.symbol.you.checked>
+   ```
+
+4. **Refuse to write `01-grill.md` until this round-trip is logged.** If
+   the cited symbol does not exist on the class, revise your snippet to
+   reference the actual shape, then re-verify and re-log.
+
+This is prompt-level discipline, not an orchestrator gate. The trace event
+provides observability so we can audit whether you skipped this step (see
+spec §"Future work" — a hard orchestrator-side gate is deferred until
+the trace shows the grill skipping verification under load).
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)